app/(authed)/dashboard/table-topics/_components/no-videos.tsx

```
@@ -0,0 +1,92 @@
+      <CardHeader>
+        <CardTitle>Embark on Your Speaking Adventure</CardTitle>
+        <CardDescription>
+          Unleash your impromptu speaking skills with Table Topics!
+        </CardDescription>
+      </CardHeader>
+      <CardContent>
+        <div className="space-y-4">
+          <p>
+            Table Topics are your gateway to mastering the art of thinking on
+            your feet. In just 1-2 minutes, transform a surprise subject into a
+            captivating mini-speech. It's not just practice—it's your personal
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)